cli/cli_test.go

```diff
@@ -575,6 +575,33 @@ func Example_sql() {
 	// 0	empty	-
 }
 
+func Example_sql_escape() {
+	c := newCLITest()
+	defer c.stop()
+
+	c.RunWithArgs([]string{"sql", "-e", "create database t; create table t.t (s string);"})
+	// We insert four rows:
+	//  - a valid printable UTF8 string
+	//  - a valid printable UTF8 string using escapes
+	//  - a valid UTF8 string with a non-printable character
+	//  - a non-UTF8 string
+	c.RunWithArgs([]string{"sql", "-e", "insert into t.t values ('κόσμε'), (e'\\xc3\\xb1'), (e'\\x01'), (e'\\xc3\\x28')"})
```

Comment:
Add `\u202e` (RTL override) as a test case (can we enter test cases with `\u` here or do we have to manually convert it to utf8? If so it's `\xe2\x80\xae`)